skymap_scanner/config.py

```diff
@@ -15,6 +15,19 @@
 
 DEFAULT_GCD_DIR: Path = Path("/opt/i3-data/baseline_gcds")
 
+# Local data sources. These are assumed to be filesystem paths and are expected to have the same directory structure.
+LOCAL_DATA_SOURCES: Final[List[Path]] = [
+    Path("/opt/i3-data"),
+    Path("/cvmfs/icecube.opensciencegrid.org/data"),
+]
+# Directory path under a local data source to fetch spline data from.
+LOCAL_SPLINE_SUBDIR: Final[str] = "photon-tablex/splines"
```

Comment:
Is this a typo? Should be `photon-tables` right?

Reply:
Indeed. It was a dirty trick to force the CI test to stage the files. I will revert it back to the correct value.

Reply:
If that's a useful test, you could use an env var (`ENV`)